src/lib/scroller.js

```diff
@@ -80,34 +88,61 @@ Scroller.prototype.requestRefresh = function(){
 Scroller.prototype._renderVisible = function(){
   this.animateRequest = null;
   if(this.dirty && this.console){
-    var top = this.console.scrollTop;
+    const top = this.console.scrollTop;
     if(this.sticky){
-      this.startPosition = Math.max(0, this.lines.length - this.minVisible);
+      this.startPosition = Math.max(this.lineOffset, this.lineCount() - this.visibleCount);
+      this.endPosition = this.lineCount();
     }
-    this.console.innerHTML = generateContent(this.lines, this.startPosition, this.lines.length, this.minVisible);
+    this.console.innerHTML = this._generateContent();
     if(this.jumpToBottom){
-      this.console.scrollTop = 2000;
+      this.console.scrollTop = 4000;
```

Comment:
The number of visible lines was increased so it wouldn't need as many expansions during a scroll, but if the font was really large it wouldn't scroll properly. It's really unlikely but I just wanted to be safe.